screenbuf/screenbuf.go

```diff
@@ -97,7 +97,7 @@ func (s *ScreenBuf) Write(b []byte) (int, error) {
 	}
 }
 
-// Flush writes any buffered data to the underlying io.Writer.
+// Flush writes any buffered data to the underlying io.Writer, essentially clearing the console.
```

Comment:
flush ensures any data that is pending to be displayed is displayed. it does a full screen repaint first, but then outputs anything that has been buffered.